### PR TITLE
Ui designtwo

### PR DESCRIPTION
**Pull request by Drishtantr**
Dummy data stored, for now, temporary admin page created, admin has access to upload, files, and transcriptions page for now.. tables updated in files and transcriptions page, other minor updates (signIn/register card, results page, etc.)

### CODE REVIEW

**Comment by shakyaprasen** [comment]
A better PR title would be preferred

File: src/assets/key1.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+The northman considered the rainbow as a bridge over which the gods passed from earth to their home in the sky.
```
Comment (shakyaprasen) [inline src/assets/key1.txt:1]: whats a txt file doing here?

File: src/components/Header.vue [hunk 2]
```diff
@@ -1,25 +1,65 @@
...
       <b-navbar-toggle target="nav-collapse"></b-navbar-toggle>
 
       <b-collapse id="nav-collapse" is-nav>
 
         <!-- Right aligned nav items -->
         <b-navbar-nav class="ml-auto">
-          <b-nav-item right>About</b-nav-item>
-          <b-nav-item right>Contact Us</b-nav-item>
+          <b-nav-item right @click="popupActivo=true">About</b-nav-item>
+          <b-nav-item right @click="popupActivo1=true">Contact Us</b-nav-item>
```
Comment (shakyaprasen) [inline src/components/Header.vue:13]: better naming for popupActivo & popupActivo1. and use a function for setting the value .

File: src/views/Files.vue [hunk 3]
```diff
@@ -1,29 +1,56 @@
...
+              {{data[indextr].avgScore}}
+            </vs-td>
+            <vs-td>
+              <vs-button @click="deleteItem(data[indextr].id)"
+              color="danger" type="line">Delete</vs-button>
+            </vs-td>
+          </vs-tr>
+        </template>
+      </vs-table>
+      <!-- <table class="table table-borderless mx-auto m-2 text-left"
```
Comment (shakyaprasen) [inline src/views/Files.vue:49]: remove unused code.

File: src/assets/key1.txt (hunk 1 above)
Comment (Drishtantr) [reply src/assets/key1.txt:1]: It is a transcription text file of an audio, we could maybe compare the result obtained with this txt file/s.

File: src/components/Header.vue (hunk 2 above)
Comment (Drishtantr) [reply src/components/Header.vue:13]: Huss, done

File: src/views/Files.vue (hunk 3 above)
Comment (Drishtantr) [reply src/views/Files.vue:49]: Huss, done

**Review by shakyaprasen — APPROVED** [review]

**Pull request merged**